packages/sil-governance/src/store/blockchain/actions.ts

```diff
@@ -0,0 +1,6 @@
+import { BlockchainState, SetBlockchainActionType } from "./reducer";
+
+export const setBlockchainAction = (blockchain: BlockchainState): SetBlockchainActionType => ({
+  type: "@@blockchain/SET",
+  payload: blockchain,
+});
```

Comment:
You can define additional action like `setBlockchainHeaderAction` here, with type: `@@blockchain/SET_HEADER`